liquidswap/cli.py

```diff
@@ -39,24 +39,34 @@ def critical(title='', message='', start_over=True):
 @click.option('-c', '--conf-file', default=None, type=str,
               help='Specify elements.conf file for authentication.')
 @click.option('-r', '--regtest', is_flag=True, help='Use with regtest.')
+@click.option('-t', '--testnet', is_flag=True, help='Use with liquidtestnet.')
```

Comment:
nit: we could use `click.Choice` and use a single flag here

    @click.option('-n', '--network', type=click.Choice(['mainnet', 'testnet', 'regtest']), default='mainnet')

We could then have use the network name directly instead of mapping to an int.

If you think it's worth it, then change it, but if you disagree feel free to ignore

Reply:
As discussed offline, we merged this and we'll work on the refactor later.